### PR TITLE
Invoice trigger

### PR DESCRIPTION
**Pull request by Dheeraj5861**

### CODE REVIEW

File: force-app/main/default/classes/InvoiceDetailTriggerHelper.cls [hunk 1]
```diff
@@ -207,6 +222,145 @@ public with sharing class InvoiceDetailTriggerHelper {
...
+    
+    }
+    
+    public Monthly_Breakdown__c calculateMonthlyBreakdownMargins(Monthly_Breakdown__c monthlyBreakdown){
+        monthlyBreakdown.Tile_Margin_Percentage__c  = ((monthlyBreakdown.Tile_Margin__c / monthlyBreakdown.Tile_Sale__c) *100).setScale(2);
+        monthlyBreakdown.Ancillary_Margin_Percentage__c  = ((monthlyBreakdown.Ancillary_Margin__c / monthlyBreakdown.ancillary_sale__c) *100).setScale(2);
+        monthlyBreakdown.Total_Invoice_Tile_Margin__c  = ((monthlyBreakdown.Tile_Margin__c  * monthlyBreakdown.Commission_Split_Percentage__c)/100).setScale(2);
+        monthlyBreakdown.Total_Invoice_Ancillary_Margin__c   = ((monthlyBreakdown.Ancillary_Margin__c  * monthlyBreakdown.Commission_Split_Percentage__c)/100).setScale(2);
+        monthlyBreakdown.Total_Margin__c = monthlyBreakdown.Total_Invoice_Tile_Margin__c + monthlyBreakdown.Total_Invoice_Ancillary_Margin__c;
+        if( monthlyBreakdown.Tile_Margin_Percentage__c >= 33.0 && monthlyBreakdown.Ancillary_Margin_Percentage__c >= 28.0){
```
Comment (Dheeraj5861) [inline force-app/main/default/classes/InvoiceDetailTriggerHelper.cls:324]: Hardcoded these values just for testing
Will perform a query and get the parameters by query result

Comment (Dheeraj5861) [inline force-app/main/default/classes/InvoiceDetailTriggerHelper.cls:324]: Hardcoded these values just for testing
Will perform a query and get the parameters by query result

Comment (RajneeshSamiteon) [reply force-app/main/default/classes/InvoiceDetailTriggerHelper.cls:324]: ok

Comment (RajneeshSamiteon) [reply force-app/main/default/classes/InvoiceDetailTriggerHelper.cls:324]: image
 some thing like this u should use to get values (once your testing is done)

**Review by RajneeshSamiteon — CHANGES_REQUESTED** [review]

**Pull request merged**